pages/linux/auracle.md

```diff
@@ -0,0 +1,20 @@
+# auracle
+
+> Auracle is a command line tool used to interact with Arch Linux's User Repository, commonly referred to as the AUR.
+> More information: <https://github.com/falconindy/auracle>.
+
+- Find packages in the AUR by regular expression:
+
+`auracle search {{regexp}}`
+
+- Print detailed information about packages:
+
+`auracle info {{packagename}} {{packagename}}`
+
+- Show the contents of a source file for a/multiple package(s) (e.g. the PKGBUILD):
+
+`auracle show {{packagename}} {{packagename}}`
```

Comment:
```suggestion
`auracle show {{package1}} {{package2}}`
```

Reply:
I'm fine with `x1, x2`, but I took it from https://github.com/tldr-pages/tldr/blob/master/pages/common/bower.md - maybe the should be a documented convention/guideline for this kind of args?

Reply:
Either wya is probably fine, but I think @navarroaxel's suggestion is slightly clearer.

We don't really have a standard convention here, but if we did it would probably be something sort of similar to @navarroaxel's suggestion. I've seen somewhere `a`, `b`, `N` used too - it doesn't really matter I don't think.